torch/testing/_internal/opinfo/definitions/linalg.py

```diff
@@ -1657,6 +1665,11 @@ def make_input():
                 device_type="mps",
                 dtypes=[torch.float32],
             ),
+            DecorateInfo(
+                unittest.skip("cpu vs gpu is sometimes the same"),
```

Comment:
NIT: Although the meaning is the same, should probably read `sometimes different`

Reply:
updated